src/version_numbers.hpp.in

```diff
@@ -0,0 +1,29 @@
+/*************************************************************************
```

Comment:
Ideally we'd like this to be in `src/cpprealm/internal` or something, so it doesn't pollute the higher header directory needlessly.